docs/spec_draft.md

```diff
@@ -1406,6 +1407,61 @@ and produces a `result` tensor. More formally,
 ```
 
 [Back to Ops](#index-of-ops)
+
+## stablehlo.rng
+
+### Semantics
+
+Constructs a `result` tensor of a given shape `shape` with random numbers
+generated using the `rng_distribution` algorithm. If `rng_distribution` $=$
+`UNIFORM`, then the random numbers are generated following the uniform
+distribution over the interval [`a`, `b`). If `rng_distribution` $=$ `NORMAL`,
+then the random numbers are generated following the normal distribution with
+mean = `a` and standard deviation = `b`.
```

Comment:
This is what HLO is doing:

https://github.com/tensorflow/tensorflow/blob/2662c525af2f99a731290657e4b494fee29feb64/tensorflow/compiler/xla/hlo/evaluator/hlo_evaluator.cc#L801-L811

which uses some predetermined config seed (default 0) or a monotonic seed per evaluator using `std::random_device`.

Reply:
Are there any guarantees that we can provide the producers about the behavior of this op? E.g. can an implementation return the same values over and over upon successive calls?

Reply:
There are no guarantees we can provide. The interpreter will provide nondeterministic values upon successive calls. I will clarify this in the spec.

Reply:
Actually, I don't think there is a need for any clarification. Since we don't provide a parameter to initialize the seed, the default should be to provide random values. Otherwise, the op is not so random :)

Reply:
I think it would be good to explicitly call out the extent of guarantees that we can provide. Proposed some wording in a nearby comment.